mllib/src/main/scala/org/apache/spark/mllib/stat/MultivariateOnlineSummarizer.scala

```diff
@@ -230,6 +230,13 @@ class MultivariateOnlineSummarizer extends MultivariateStatisticalSummary with S
   override def count: Long = totalCnt
 
   /**
+   * Number of parameters
+   *
+   */
+  @Since("2.3.0")
+  override def numParam: Int = n
```

Comment:
`numParams`

Reply:
Hmmm... no this isn't right. When this is used with regression metrics, the summary is of dimension 2 always since the `predictionsAndObservations` is just an rdd of (prediction, label) pairs. The unit tests weren't thorough enough to catch it.